aligner/src/main/java/org/omegat/gui/align/AlignFilePickerController.java

```diff
@@ -523,7 +524,9 @@ public static void main(String[] args) throws Exception {
         System.setProperty("apple.laf.useScreenMenuBar", "true");
 
         Preferences.init();
+        UIDesignManager.initialize();
```

Comment:
Current Main class load plugin before calling `Core.initializeGUI`.

Here is a copy from Main class
```
        Preferences.init();
        // broker should be loaded before module loading
        JLanguageTool.setClassBrokerBroker(new LanguageClassBroker());
        JLanguageTool.setDataBroker(new LanguageDataBroker());
        PluginUtils.loadPlugins(PARAMS);
        FilterMaster.setFilterClasses(PluginUtils.getFilterClasses());
        Preferences.initFilters();
        Preferences.initSegmentation();
        
        // then call Core.initializeGUI
       // InitializeGUI call UIDesignManager.initialize()
```

Call it in the timing before `loadPlugins`, the `UIDesignManager` does not find theme plugins, escpecially for default theme.